calcom/cal.com
# add meeting url to dynamic text variables of custom workflow templates

## What does this PR do?

Adds {MEETING_URL} to the dynamic text variables in workflows. 

Loom Video: https://www.loom.com/share/4bd66feaf84e427e9b38af1fa3004e2d

Fixes #5421

## How should this be tested?

- Create a workflow that triggers when a new event is booked 
- Add a custom email or SMS body with the {MEETING_URL} in it 
- Book an event type that has a video conferencing tool as the location
- Check the sent workflow SMS/Email, it should include the video conferencing link
